### PR TITLE
Scaffold single obra

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: lib/meta-boxes.php [hunk 1]
```diff
@@ -98,8 +98,15 @@ function igv_cmb_metaboxes() {
...
+    'name'       => __( 'Medium ES', 'cmb2' ),
+    'desc'       => __( 'en español', 'cmb2' ),
+    'id'         => $prefix . 'medium_es',
+    'type'       => 'text',
+  ) );
+
+  $obra_meta->add_field( array(
+    'name'       => __( 'Medium EN', 'cmb2' ),
+    'desc'       => __( 'in english', 'cmb2' ),
     'id'         => $prefix . 'medium',
```
Comment (mixcas) [inline lib/meta-boxes.php]: why is this not postfixed? is this the default lang?

File: single-obra.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+              if (!empty($meta['_igv_year'])) {
+                echo '<br/>' . $meta['_igv_year'][0];
+              }
+            ?>
+          </h2>
+
+          <div id="single-work-meta">
+            <?php
+              if (!empty($meta['_igv_medium_es']) && !empty($meta['_igv_medium'])) {
+                echo '<div>' . __('[:es]' . $meta['_igv_medium_es'][0] . '[:en]' . $meta['_igv_medium'][0]) . '</div>';
```
Comment (mixcas) [inline single-obra.php]: if we are going to be doing the logic of which field to echo depending on the lang, we should follow some pattern. 
i think `qtranxf_getLanguage()` would return the lang code, we could use this code for postfix the meta id's
that way and have a simpler logic and simpler outputs

File: lib/meta-boxes.php (hunk 1 above)
Comment (varffs) [inline lib/meta-boxes.php]: ah yh fair enough

File: single-obra.php (hunk 2 above)
Comment (varffs) [inline single-obra.php]: i dunno tho. why is that better than localising a string? I suppose it would do more or less the same either way

Comment (mixcas) [inline single-obra.php]: I feel it is a better pattern in gral

```
          <div id="single-work-meta">
            <?php

              if (!empty($meta['_igv_medium_' . $lang])) {
                echo '<div>' . __($meta['_igv_medium_' . $lang])[0] . '</div>';
              }
              if (!empty($meta['_igv_size'])) {
                echo $meta['_igv_size'][0];
              }
            ?>
          </div>
```

Comment (varffs) [inline single-obra.php]: yeah far enuf i'll change it

**Comment by varffs** [comment]
@mixcas ready to close i think

**Comment by mixcas** [comment]
so beautiful

**Pull request merged**